### PR TITLE
Identify unsafe stubs in javadocs

### PR DESCRIPTION
**Pull request by theresa-m**
Signed-off-by: Theresa Mammarella <Theresa.T.Mammarella@ibm.com>

### CODE REVIEW

File: runtime/jcl/common/sun_misc_Unsafe.cpp [hunk 1]
```diff
@@ -159,6 +159,8 @@ Java_sun_misc_Unsafe_pageSize(JNIEnv *env, jobject receiver)
 jint JNICALL
 Java_sun_misc_Unsafe_getLoadAverage(JNIEnv *env, jobject receiver, jdoubleArray loadavg, jint nelems)
 {
+	/* stub implementation */
+	assert(!"Java_sun_misc_Unsafe_getLoadAverage is unimplemented");
```
Comment (theresa-m) [inline runtime/jcl/common/sun_misc_Unsafe.cpp]: I'm assuming this assert was forgotten at some point?

Comment (pshipton) [reply runtime/jcl/common/sun_misc_Unsafe.cpp]: Do you mean did we forget to implement this? OpenJDK uses the unsafe API to implement OperatingSystemMXBean.getSystemLoadAverage(). OpenJ9 has a separate implementation of OperatingSystemMXBean, and implements this without calling unsafe, using the OpenJ9 native getSystemLoadAverageImpl instead. Nobody has ever complained about the unsafe API not working, but it should be possible to implement using the existing OpenJ9 code as a basis, for a single element array. The parameters aren't exactly the same between the two implementations.

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8

File: runtime/jcl/common/sun_misc_Unsafe.cpp (hunk 1 above)
Comment (pshipton) [reply runtime/jcl/common/sun_misc_Unsafe.cpp]: Guess I misunderstood the original comment, but there you go.

**Comment by pshipton** [comment]
jenkins test extended plinux jdk8
jenkins test all plinux jdk9

**Comment by pshipton** [comment]
jenkins test extended plinux jdk8

**Pull request merged**